config-tool/SAMLBridgeConfiguration/obj/SAMLBridgeConfiguration.csproj.FileListAbsolute.txt

```diff
@@ -0,0 +1,9 @@
+C:\_tag\google-saml-bridge-for-windows\Config-Tool\SAMLBridgeConfiguration\obj\Release\SAMLBridgeConfiguration.exe

```

Comment:
This file doesn't seem to be reasonably versionable. Is it created automatically, or do we really need it as an input? From your previous branch, the Debug directory was removed, from the Release directory is still here. Is that expected? Finally, the change in case isn't mirrored here, so it might be worth just editing this file by hand to change the case, since the file system itself is case-insensitive. Worth looking for other files that preserve the mixed-case "Config-Tool".